src/lib/config.js

```diff
@@ -1,5 +1,11 @@
 require("dotenv").config();
 
+// These are the suffixes that the HMPO framework adds to the ID of a date
+// control to generate the IDs of the text inputs.
+const HMPO_SUFFIX_DAY = "-day";
```

Comment:
Technically this is a requirement by the [Date Input field in the Design System](https://design-system.service.gov.uk/components/date-input/)

```
<div class="govuk-form-group">
          <label class="govuk-label govuk-date-input__label" for="passport-issued-day">
            Day
          </label>
          <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="passport-issued-day" name="passport-issued-day" type="text" inputmode="numeric">
        </div>
```